framework/console/controllers/BaseMigrateController.php

```diff
@@ -52,7 +52,7 @@
      *
      * @see $migrationNamespaces
      */
-    public $migrationPath = '@app/migrations';
+    public $migrationPath = ['@app/migrations'];
```

Comment:
applied this change in https://github.com/yiisoft/yii2/pull/14241/commits/844eacba26f392b5fbc2e491c256634e5ce25faa